### PR TITLE
Moving text to locales file

### LINKED ISSUE #21: Utilize internationalization (i18n)
It would be a good practice to use i18n to replace hard-coded texts and make html files more readable. And this would also allow us to support multiple languages in the future for different organizations.

### PR DESCRIPTION
**Pull request by nsiwnf**
### Describe your change in plain English.
Extracting hard-coded text into variables to allow for translations (in smaller commits for easier code-review).
Once translations are in, we can include `?locales=es` to see the translations.

In this PR
- rails i18n gem
- adoptable_pets
- organization_pets

### Link to the issue
Resolves #21

### CODE REVIEW

File: app/views/adoptable_pets/show.html.erb [hunk 1]
```diff
@@ -114,47 +102,47 @@
                   <% if AdopterApplication.adoption_exists?(current_user.adopter_account.id, @pet.id) %>
                     <div class='d-flex align-items-center mt-3'>
                       <h4 class="me-2">
-                        Application <%= @adoption_application.status.titleize || " submitted" %>
+                        <%= "#{t(".status." + @adoption_application.status)}" %>
```
Comment (Thrillberg) [inline app/views/adoptable_pets/show.html.erb:105]: This dot-accessing variable thing is so cool!

Comment (Thrillberg) [reply app/views/adoptable_pets/show.html.erb:105]: Oh but did we leave out the conditional where it can have `" submitted"`?

File: app/views/organization_pets/_form.html.erb [hunk 2]
```diff
@@ -67,38 +67,38 @@
...
+
         <% pet.errors.full_messages_for(:images).each do |message| %>
           <div class="alert alert-danger mt-1" role="alert"><%= message %></div>
         <% end %>
       </div>
 
       <div>
         <% if pet.images.attached? %>
-          <p>Uploaded Images</p>
+          <p><%= t('.upload_images') %></p>
```
Comment (Thrillberg) [inline app/views/organization_pets/_form.html.erb:94]: Another nit but maybe change this variable name to `.uploaded_images`?

File: config/locales/en.yml [hunk 3]
```diff
@@ -31,3 +31,85 @@
...
+        paused_until_further_notice: Applications Paused Until Further Notice
+      apply_to_adopt: "Apply to Adopt"
+      complete_your_profile: "Complete your profile to apply for this pet"
+      complete_my_profile: "Complete my profile"
+      adopt: "Adopt"
+      login: "Log in"
+  organization_pets:
+    show:
+      pause_applications: "Pause Applications?"
+      why: "Why?"
```
Comment (Thrillberg) [inline config/locales/en.yml:92]: I celebrate any variable which is called `why`. 🎉

**Review by Thrillberg — APPROVED** [review]
This is so rad! Thanks @nsiwnf ! I have some comments in here but I don't think any of them are blocking so feel free to ignore. 😄

File: app/views/adoptable_pets/index.html.erb [hunk 4]
```diff
@@ -25,16 +25,16 @@
...
                     <% if pet.has_adoption_pending? %>
-                      <%= pet.name %> (Adoption Pending)
+                      <%= "#{pet.name} #{t('general.adoption_pending')}" %>
                     <% else %>
                       <%= pet.name %>
                     <% end %>
                   </h3>
                 <p class="card-text m-0 bigger">
-                  Age: <%= time_ago_in_words(pet.birth_date) %>
+                  <%= "#{t('general.age_colon')} #{time_ago_in_words(pet.birth_date)}" %>
```
[the remark this answers is not in the document]
Comment (nsiwnf) [reply app/views/adoptable_pets/index.html.erb:34]: Yeah thats a good point

File: app/views/adoptable_pets/show.html.erb (hunk 1 above)
Comment (nsiwnf) [reply app/views/adoptable_pets/show.html.erb:105]: Yeah, I dont think thats a condition we'll realistically run into, I can add it back in just in case
